Something/src/test/java/org/example/NameListsTest.java

```diff
@@ -0,0 +1,39 @@
+package org.example;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class NameListsTest {
+    NameLists nameLists;
+    @BeforeEach
+    public void setUp(){
+        nameLists = new NameLists();
+        nameLists.setNamesA("John");
+        nameLists.setNamesA("Brian");
+        nameLists.setNamesA("Steve");
+
+        nameLists.setNamesB("Brian");
+        nameLists.setNamesB("Barry");
+        nameLists.setNamesB("Tyrone");
+        nameLists.setNamesB("Hector");
+        nameLists.setNamesB("Jimmy");
+    }
+    @Test
+    public void testReturnNamesA(){
+        nameLists.returnNamesA();
+    }
+    @Test
+    public void testReturnNamesB(){
+        nameLists.returnNamesB();
+    }
+    @Test
+    public void testGetNamesAND() {
```

Comment:
From the above advice, you might have changes the method being tested to getNamesAAndNamesB() 

So typically in a test, you assign the results of the method being tested to a local variable e.g.

List<String> actual =  getNamesAAndNamesB() 

then use asserts to interrogate the results, try this question in chatgpt

In junit how do I user hamcrest asserts to the results

Once you have added in some assertThat methods, give me a shout and we can re-review.